Library/Homebrew/cmd/install.rb

```diff
@@ -183,18 +204,41 @@ def self.install
     end
 
     if casks.any?
-      Cask::Cmd::Install.install_casks(
-        *casks,
-        binaries:       args.binaries?,
-        verbose:        args.verbose?,
-        force:          args.force?,
-        adopt:          args.adopt?,
-        require_sha:    args.require_sha?,
-        skip_cask_deps: args.skip_cask_deps?,
-        quarantine:     args.quarantine?,
-        quiet:          args.quiet?,
-        dry_run:        args.dry_run?,
-      )
+      quarantine = args.quarantine?.nil? ? true : args.quarantine?
```

Comment:
```suggestion
      quarantine = args.quarantine?.nil? ? true : args.quarantine?
```

I'm not sure `args.quarantine?.nil?` will ever be `true`, can you confirm?

Reply:
You are right, "args.quarantine?" is always "true" even if we don't pass the quarantine option.

Reply:
`--[no-]` options are triple state: true, false or nil (not specified). Mostly because the default of not specifying is not always static.

Reply:
I assume the quarantine option only has two `true/false` values based on this line of code: https://github.com/Homebrew/brew/blob/c46ee85e0a992fffd555e546c5c1aa911619b4e0/Library/Homebrew/env_config.rb#L488